src/components/calculator.js

```diff
@@ -1,41 +1,50 @@
     const yellowButton = `${uniformStyles} bg-warning`;
     return (
       <div className="row border w-50">
-        <div className="col-12 div height border d-flex justify-content-end align-items-end bg-secondary text-light">0</div>
-        <div className={uniformStyles}>AC</div>
-        <div className={uniformStyles}>+/-</div>
-        <div className={uniformStyles}>%</div>
-        <div className={yellowButton}>÷</div>
-        <div className={uniformStyles}>7</div>
-        <div className={uniformStyles}>8</div>
-        <div className={uniformStyles}>9</div>
-        <div className={yellowButton}>X</div>
-        <div className={uniformStyles}>4</div>
-        <div className={uniformStyles}>5</div>
-        <div className={uniformStyles}>6</div>
-        <div className={yellowButton}>-</div>
-        <div className={uniformStyles}>1</div>
-        <div className={uniformStyles}>2</div>
-        <div className={uniformStyles}>3</div>
-        <div className={yellowButton}>+</div>
-        <div className="col-6 div height border d-flex justify-content-center align-items-center">0</div>
-        <div className={uniformStyles}>.</div>
-        <div className={yellowButton}>=</div>
+        <div className="col-12 div height border d-flex justify-content-end align-items-end bg-secondary text-light">{this.state.next || this.state.total}</div>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>AC</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>+/-</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>%</button>
+        <button type="button" className={yellowButton} onClick={this.clickHandler}>÷</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>7</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>8</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>9</button>
+        <button type="button" className={yellowButton} onClick={this.clickHandler}>x</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>4</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>5</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>6</button>
+        <button type="button" className={yellowButton} onClick={this.clickHandler}>-</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>1</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>2</button>
+        <button type="button" className={uniformStyles} onClick={this.clickHandler}>3</button>
+        <button type="button" className={yellowButton} onClick={this.clickHandler}>+</button>
```

Comment:
- [ ] When i clicked an operator button consecutively more than once, I received this browser error `Error: [big.js] Invalid number`. KIndly implement an error handling to take care of scenarios like this.
- [ ] The `AC` button currently shows a blank screen when clicked instead of `0`. Please, let's fix it.

Reply:
Thanks for your feedback. All this has been resolved